src/components/ebay-icon-button/index.marko

```diff
@@ -30,7 +30,9 @@ $ {
         input.class,
         'icon-btn',
         isBadged && `icon-btn--badged`,
-        input.transparent && 'icon-btn--transparent'
+        input.transparent && 'icon-btn--transparent',
+        input.size === 'small' && 'icon-btn--small',
+        input.size === 'large' && 'icon-btn--large'
```

Comment:
We could do it like this:
```
$ const validSizes = ['small', 'large'];
```
And then:
```
validSizes.indexOf(input.size) > -1 && `icon-btn--${input.size}`
```
Just so we don't have too many of these statements, or if we need to add a new size its fine

Reply:
Nice refactor, thanks. I've addressed all of your changes.